packages/render-fidelity-tools/src/common.ts

```diff
@@ -26,6 +26,10 @@ export const MAX_COLOR_DISTANCE: number = 35215;
 
 export const DEVICE_PIXEL_RATIO: number = 2;
 
+// using this threshold to do automatic fidelity test for model-viewer. any
+// scenario whose rms value (in dB) is bigger than the threshold will fail.
+export const FIDELITY_TEST_THRESHOLD: number = -18;
```

Comment:
im just using an arbitrary threshold here.  the highest rms in dB in dev page is -24 (assuming it's right), i guess probably -20 would be a reasonable threshold?